types/trace/trace.go

```diff
@@ -40,6 +40,7 @@ type Event struct {
 	ReturnValue         int          `json:"returnValue"`
 	StackAddresses      []uint64     `json:"stackAddresses"`
 	ContextFlags        ContextFlags `json:"contextFlags"`
+	OriginatedSyscall   int          `json:"originatedSyscall"`
```

Comment:
I'm conflicted about moving this here but if we go with this then:
1. Please fix the syscall argument filtering and move it to the context filter
2. style: I think this fits slightly better above `ReturnValue` and that just `Syscall` is enough.
3. Previously when `parse-arguments` was used, the syscall would be translated to it's string form, making it readable. This is no longer the case, and will not be present in the table printer.
4. To solve the above pointer, and keep syscalls normalized, I think we should change this to `string` and process it to a syscall name in `processEvent`.

Reply:
1. I prefer something else than "OriginatedSyscall"
2. I agree that this should be normalized to a string.